docs/ingestion/ingestion-spec.md

```diff
@@ -472,24 +474,23 @@ An example `ioConfig` to read JSON data is:
 }
 ```
 
-For details, see the documentation provided by each [ingestion method](./index.md#ingestion-methods).
-
 ## `tuningConfig`
 
-You specify tuning properties in a `tuningConfig` object, which goes at the top level of an ingestion spec.
-Some properties apply to all [ingestion methods](./index.md#ingestion-methods), but most are specific to each individual ingestion method.
+Specify tuning properties in a `tuningConfig` object at the top level of an ingestion spec.
```

Comment:
```suggestion
You specify tuning properties in a `tuningConfig` object at the top level of an ingestion spec.
```
I think it reads better with "you" since it's a conceptual topic and not a tutorial.